src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Should this be a wildcard import?
From [SE-Guide](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements):  Imported classes should always be listed explicitly.